operator/go.mod

```diff
@@ -28,6 +28,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/baidubce/bce-sdk-go v0.9.141
+
```

Comment:
We don't need this import because the import reference below is wrong.

Reply:
Done